examples/tools/system/package_manager.rst

```diff
@@ -0,0 +1,200 @@
+.. _examples_tools_system_package_manager:
+
+System Packages
```

Comment:
I think I'd use a more descriptive title, like ``Wrapping system requirements in a Conan package``.

Then, it would be great to discuss a bit more the problem/solution:

- Describe the problem that system_requirements by default execute
- At the end, describe how this approach avoids system_requirements() to be skipped and not run, if not needing to build


It would also be good, in a small subsection, to discuss the possibility of running the ``Apt()`` commands inside ``build()`` and cite pros and cons compared with the Conan wrapper recipe.

Reply:
> It would also be good, in a small subsection, to discuss the possibility of running the Apt() commands inside build() and cite pros and cons compared with the Conan wrapper recipe.

I would prefer keeping it in a next and separate PR, because there is a specific issue: https://github.com/conan-io/docs/issues/3975 So we can have a smaller PR here.

Reply:
@memsharded I updated the title on the commit [79f9fe1](https://github.com/conan-io/docs/pull/3978/commits/79f9fe196a15c7af54081998ebb60573f28f379d) and added a note about system requirements running always on the commit [4e4246e](https://github.com/conan-io/docs/pull/3978/commits/4e4246e0a93e386c3e47f9904fb7a47df8acae50)